webservices/schemas.py

```diff
@@ -50,13 +49,13 @@ def __new__(cls, field, column, label, position=-1):
         )
 
 
-class BaseSchema(ModelSchema):
```

Comment:
ModelSchema has been removed from marshmallow-sqlalchemy. We must either use SQLAlchemySchema or SQLAlchemyAutoSchema. 

Looking a the [documentation](https://marshmallow-sqlalchemy.readthedocs.io/en/latest/index.html#generate-marshmallow-schemas) we might have to explicitly declare all fields for a schema. We can try to use the  auto_field() method. But we would still need to declare the names of the fields